planning/development-strategy.md

```diff
@@ -1,39 +1,142 @@
-# Restaurants finder
+# Job - Advisor
+
+<!---minju edited--->
+
+Job Advisor website is for motivated students and job seekers to help them to
+kickstart their career in IT and also search their dream job without
+registration.
 
 ---
 
-<!-- ## User Story Dependencies
+## Setup
```

Comment:
@minjupgeorge It would be nice to add React, Mui, Strapi

Reply:
ok I will update it